boards/st/stm32h750b_dk/stm32h750b_dk.yaml

```diff
@@ -15,4 +15,6 @@ supported:
   - flash
   - rtc
   - memc
+  - display
+  - spi
```

Comment:
maybe also 
 ` - qspi`

Reply:
@FRASTM There is no test/test case in-tree that uses the `qspi` tag & concerns this board.